.gitattributes

```diff
@@ -0,0 +1,10 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
```

Comment:
This comment does not reflect what's happening here.

Reply:
Does the suggestion below sound ok?

Reply:
Looks good to me!